### DISCUSSION TITLE
Can the calculation between the modules be done in parallel?

### DISCUSSION THREAD

**Discussion by Yutsky006** [comfyanonymous/ComfyUI#2693] [Q&A]
Want to ask if comfyui has the ability to parallel requests. For example, VAE and UNet can be deployed in different places, with http requests.

**Comment by ltdrdata** [comment]
Unfortunately, ComfyUI currently does not support parallel processing.

But take a look around this
https://github.com/city96/ComfyUI_NetDist
https://github.com/city96/LiliumSD
